### ISSUE TITLE
Improve article : 2308.06245v1

### ISSUE THREAD

**Issue by Shantanu42** [arXiv/html_feedback#103]
## Describe the issue

**Description**: Thank you very much for doing this. I really like the black background and I am extremely excited!
The issue I notice might just be a Latex quark. In some of the lines that proceed an equation the spacing goes wider and makes it hard to follow.

Feel free to attach a screenshot (or document) link below: 

## Auto Fill Data - !!! Please do not edit below this line !!!
----------------------------------------------------------------------------------------

Id: 20fdac89-e967-4563-83ca-dc17f69c549c

**Comment by dginev** [comment]
Thank you for your report - the specific issue is now solved.

We have regenerated the article and you should be able to see an improvement.
